Dojo/a very big sum

adrianoavelino opened the pull request:
Código gerado durante o dojo do dia 25/11/2017 utilizando a linguagem PHP utilizando o desafio do [HackerRank](https://www.hackerrank.com/challenges/a-very-big-sum/problem)

adrianoavelino committed:
Inicia desafio do dia 25/11/2017 (PHP)

Criar projeto "A Very Big Sum" com testes e composer configurados e rodando.

adrianoavelino committed:
Refatora a função e inicia teste do item do array

Refatora a função adicionando uma condição if para verificar o tamanho
do array e inicia um teste para verificar o valor mínimo e máximo o
valor de cada array pode receber

lflimeira reviewed (changes requested):
@adrianoavelino , faltou aquela parte de pontos positivos e negativos e o que foi aprendido no dojo. Pode adicionar isso pf? 😃

adrianoavelino commented:
Então, na verdade o desafio nem foi concluído. O Maicon combinou algo de terminar outro dia. Acho que temos que definir isso com o pessoal que particiou

lflimeira commented:
Entendo @adrianoavelino, mas o Dojo n precisa terminar. Lembra na semana passada que fizemos mesmo sem o Dojo acabar e colocamos como status: incompleto. Dessa forma outras pessoas podem tentar terminar depois. 😃

adrianoavelino committed:
Adiciona README do dojo

Adiciona um arquivo README com as informações do desafio e descrive os
pontos positivos e negativos do Dojo.

adrianoavelino commented:
Adicionado um README dentro do desafio

woliveiras commented:
@lflimeira tudo ok aqui, meu querido?

lflimeira reviewed (approved):
Tudo certo @woliveiras, desculpa a demora.

Obrigado pelas alterações @adrianoavelino =)
